### ISSUE TITLE
Wrong Quest Reward for Druid of the Claw Quest

### ISSUE THREAD

**Issue by ForlornHunter** [LightsHope/issues#611]
The level 10 Druid of the Claw quest started by Oben Rageclaw in Teldrassil has the incorrect quest reward (check wowhead or other such sources):

Currently you are able to choose either the Sleeping Robes or **Two-Handed Brushwood Blade** Sword with 7.2 DPS. The correct quest reward for the Sword should be the **One-Handed Brushwood Blade** Sword with 5.3 DPS.

Night Elves cannot even use Two-Handed Swords until they travel to Stormwind to train them. Can we please have this corrected as it is the first decent Green Weapon for NE Warriors to obtain.

**Comment by Gamemechanicwow** [comment]
There is both a 1h(18957) and 2h(9602) item in databases - allakazam etc. 

Patch 1.8  - The Brushwood Blade item tooltip has been updated to indicate that it is a two-handed sword, not one-handed. 

From what i can tell, it was originally a 1h (bugged) that got fixed to 2h in 1.8 and later(not sure when) replaced by a 1h.

**Comment by ForlornHunter** [comment]
Yes, both of those items are in the databases however only the One-Handed is the correct reward for the Druid of the Claw quest. Instead of looking up the items, look up the quest to see what the rewards are listed as.

As I said in my OP, NE's can't even use Two-Handed Swords unless they travel to Stormwind to train them so why would a level 10 quest in the NE starting zone offer one as a quest reward?

**Comment by Gamemechanicwow** [comment]
sigh.. I was just sharing the research i had done. Since this item/quest seems to have had some changes thats relevant for a progressive server.

As a short follow up, i can say that the 18 patchnotes i referenced are not the official, but from the ptr.
http://www.tomshardware.co.uk/forum/36195-13-patch-notes
"The Brushwood Blade item tooltip has been updated to indicate that it is a two-handed sword, not one-handed."

Official patchnotes from 1.5:
"◾The Druid of the Claw quest in Teldrassil will now provide a weapon choice more appropriate for low level Night Elves. "
So, it seems like that pre 1.5 it wasn't a 1h sword.

> As I said in my OP, NE's can't even use Two-Handed Swords unless they travel to Stormwind to train them so why would a level 10 quest in the NE starting zone offer one as a quest reward?

I understand your logic, but looking at quests in Teldrassil there is also Crown of the earth and Ursal the mauler. And im sure there is several similar quests in other starting areas.

**Comment by ForlornHunter** [comment]
@stesve I wanted to apologize for my last comment as, after rereading it, it sounds as if I am slamming your previous comment and getting short with you. That was not my intention at all. I was just trying to convey my findings in the shortest TLDR I could and failed in the politeness department.

Hopefully once we hit the 1.5 patch the reward will change and we can see if you are correct in all of your research (which I do appreciate you doing). =)
